core/dbt/parser/manifest.py

```diff
@@ -749,12 +743,42 @@ def _process_docs_for_exposure(
 # exposures: exposure descriptions
 def process_docs(manifest: Manifest, config: RuntimeConfig):
     for node in manifest.nodes.values():
-        ctx = generate_runtime_docs(
-            config,
-            node,
-            manifest,
-            config.project_name,
-        )
+        if isinstance(node, ParsedSchemaTestNode):
+            # Idealing we should be able to put the test kwargs in the context,
+            # but kwargs have already been processed at this point
+            # ex: 'model' --> "{{ ref('model') }}"
```

Comment:
Just to clarify my understanding: The `test_metadata` that lives on the test node, and contains `kwargs` as well, is _not_ Jinja-rendered. E.g:
```json
"test_metadata": {
    "name": "accepted_values",
    "kwargs": {
        "values": [
            "usd"
        ],
        "column_name": "currency_code",
        "model": "{{ ref('stg_ticket_tailor__orders') }}"
    },
    "namespace": null
},
```
In order to include `kwargs` in the test description, we'd need to upgrade the `description` renderer to use the _full_ Jinja rendering context (refs, macros, all that). Is that right?